docs/docs/components/view.md

```diff
@@ -73,7 +73,7 @@ restrictFocusWithin: boolean = false; // web only
 // the arrow keys (or using some other behaviour).
 // WARNING: For the sake of performance, this property is readonly and
 // changing it during the View life cycle will produce an error.
-limitFocusWithin: boolean = false; // web only
+limitFocusWithin?: LimitFocusType = undefined; // web only
```

Comment:
Please add documentation for the LimitFocusType so it's clear what the three different enumeration states mean.